partner_manual_rank/tests/test_res_partner.py

```diff
@@ -43,6 +44,16 @@ def test_01_is_customer(self):
                 {"is_customer": False, "customer_rank": 0},
             ],
         )
+        partners.write({"customer_rank": 1})
+        self.assertRecordValues(
+            partners,
+            [
+                {"is_customer": True, "customer_rank": 1},
+                {"is_customer": True, "customer_rank": 1},
+            ],
+        )
+        with self.assertRaises(UserError):
```

Comment:
Use

`self.assertRaisesRegex(UserError, error_msg):`

To ensure the raised error is the expected one.

Which one are you expecting here?

Same below.

Reply:
Hi, I added this to cover these function

![image](https://github.com/OCA/partner-contact/assets/151717349/84c427d5-41e3-4d63-a631-985e60569444)

I just updated the code to use `assertRaisesRegex`